packages/@lwc/babel-plugin-component/src/__tests__/fixtures/decorators/unnecessary-register-decorators/latest/should-register-decorators-for-anonymous-class-that-extends-non-lightning-element/actual.js

```diff
@@ -0,0 +1,8 @@
+import { api } from "lwc";
+import MyCoolMixin from './mixin.js'
+
+const foo = class extends MyCoolMixin {
```

Comment:
Do you only fail when it doesn't extend anything, since it couldn't possibly be extending `LightningElement` indirectly?

Reply:
Exactly. `extends` is a sniff for "might be a LightningElement."